docker-compose.yml

```
@@ -3,8 +3,8 @@ version: '3.8'
 services:
   mailserver:
     image: docker.io/mailserver/docker-mailserver:latest
-    hostname: <HOSTNAME> # <-- CHANGE THIS
-    domainname: <DOMAINNAME> # <-- CHANGE THIS
+    hostname: mailserver # <-- CHANGE THIS
+    domainname: localhost # <-- CHANGE THIS
```

```suggestion
    hostname: mail # <-- CHANGE THIS
    domainname: example.com # <-- CHANGE THIS
```
https://github.com/docker-mailserver/docker-mailserver/pull/2159/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R7

---

Hold off for a few minutes on a review. I have to revert a bunch of stuff